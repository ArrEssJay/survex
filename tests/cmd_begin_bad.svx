; pos=fail warn=0 error=3
; We used to report "End of line not blank" but 1.2.22 gives "Character "=" not
; allowed in station name (use *SET NAMES to set allowed characters)" instead.
*fix 1 reference 0 0 0
*begin *
*end
*begin .
*end
*begin =
*end
