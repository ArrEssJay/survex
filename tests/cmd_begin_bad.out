./cmd_begin_bad.svx:5:8: error: Character "*" not allowed in station name (use *SET NAMES to set allowed characters)
./cmd_begin_bad.svx:7:8: error: Character "." not allowed in station name (use *SET NAMES to set allowed characters)
./cmd_begin_bad.svx:9:8: error: Character "=" not allowed in station name (use *SET NAMES to set allowed characters)

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
There were 0 warning(s) and 3 error(s) - no output files produced.
